app/views/evidence/_evidence.html.erb

```diff
@@ -1,11 +1,21 @@
-<%= content_tag :div, id: "#{dom_id(evidence)}_link", class: ('list-item' << ' active' if (request.format.html? and @evidence == evidence)) do %>
+<%
+  item_css = ['list-item']
+  item_css << ' active' if (request.format.html? and @evidence == evidence)
+%>
+
+<%= content_tag :div, id: "#{dom_id(evidence)}_link", class: item_css.join do %>
   <div class="expansion-container">
+    <%= link_to edit_node_evidence_path(@node, evidence), class: 'list-item-edit' do %>
+      <i class="fa fa-pencil"></i> Edit
+    <% end %>
     <%= link_to node_evidence_path(@node, evidence) do %>
-      <% if tag = evidence.issue.tags.first %>
-        <i class="fa fa-flag" style="color: <%= tag.color %>"></i>
-      <% else %>
-        <i class="fa fa-flag"></i>
-      <% end %>
+      <%
+        options = { class: 'fa fa-flag list-item-icon' }
+        if tag = evidence.issue.tags.first
```

Comment:
I'm not a fan of assignments in conditions because of the readability. I know that this can be found all over the codebase, but I think we should reduce the usage of this. What do you think?